src/common/utils/pagination.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  paginate,
+  PaginationArgs,
+  PaginationOptions,
+} from 'nestjs-prisma-pagination';
+
+type PrismaDeLegate<E> = {
+  count: (args?: any) => Promise<number>;
+  findMany: (args?: any) => Promise<E[]>;
+};
+
+interface ExtendedPaginationArgs extends PaginationArgs {
+  search?: string;
+}
+// Esses dois para incluir o search neles, para não dar erro de tipo.
+interface ExtendedPaginationOptions extends PaginationOptions {
+  search?: string[];
+  disableInsensitiveMode?: boolean;
+}
+
+export async function Paginate<
```

Comment:
Nome de função inicia com letra minúscula.
```suggestion
export async function paginate<
```